e2e/app/modal/share-modal.ts

```diff
@@ -99,4 +117,19 @@ export default class ShareModal extends Modal {
     const levelText = level[0].toUpperCase() + level.substring(1).toLowerCase();
     return this.page.waitForXPath(`//*[starts-with(@id,"react-select") and text()="${levelText}"]`, { visible: true });
   }
+
+  async notFindUser(email: string): Promise<boolean> {
```

Comment:
where&how this method is used?

Reply:
not used yet. I plan to use it in future in a new negative search test.